8.x/core-concepts/validation.md

```diff
@@ -2,4 +2,393 @@
 
 [[toc]]
 
-Should be filled.
\ No newline at end of file
+This document talks about checking the validity of requests that are sent from the client side to the server. The Larammerce platform is written based on the Laravel framework, so it uses the same validation rules with some modifications to make it easier.
+
+### Validation in Laravel framework
+
+You can study the official Laravel documentation to learn more about this topic.*<sup>[1](#1)</sup>*
```

Comment:
As you allocated a segment for the validation in the Laravel framework, I think it's better to write a line or two and then refer readers to the main website. but still, if you think it's not necessary I accept. it's not a problem but a way to improve this section.